src/java/com/unifina/signalpath/blockchain/ContractEventPoller.java

```diff
@@ -77,30 +86,41 @@ private void newFilter() {
 		}
 		log.info(String.format("Filter '%s' created. Listening to contract '%s' on canvas '%s'.",
 			filterId, contractAddress, id));
-	}
 
-	/**
-	 * https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getfilterchanges
-	 */
+	}
 	private synchronized void pollChanges() {
-		log.info(String.format("Polling filter '%s'.", filterId));
+		if(filterId == null){
```

Comment:
Please pay attention to formatting. IDEA is your friend here:
- Select method
- Code -> Reformat code